src/rmrk2.0.0/tools/consolidator/interactions/mint.ts

```diff
@@ -23,6 +23,14 @@ export const validateMintNFT = async (
       `Attempted issue of NFT in non-owned collection. Issuer: ${nftParentCollection.issuer}, caller: ${remark.caller}`
     );
   }
+  
+  const nfts = await dbAdapter.getNFTsByCollection(nftParentCollection.id);
+  const unburnedNfts = nfts ? nfts.filter((nft) => nft.burned === "") : [];
+  if (nftParentCollection.max === unburnedNfts.length && nftParentCollection.max !== 0) {
```

Comment:
I would probably do `unburnedNfts.length >= nftParentCollection.max`

Reply:
Ok changed it